src/platform/site-wide/announcements/config/index.js

```diff
@@ -22,10 +23,18 @@ const config = {
       disabled: !WelcomeToNewVAModal.isEnabled(),
     },
     {
-      name: 'find-va-benefits-intro',
+      name: 'dashboard-intro',
+      paths: /^(\/my-va\/)$/,
+      component: DashboardIntro,
+      disabled: !environment.isProduction(),
+      relatedAnnouncements: ['personalization'],
+    },
+    {
+      name: 'find-benefits-intro',
       paths: /^(\/my-va\/)$/,
       component: FindVABenefitsIntro,
       disabled: environment.isProduction(),
+      relatedAnnouncements: ['personalization'],
```

Comment:
It seems like we will want to have the same related announcements for the find benefits announcement, as for the dashboard intro announcement.

Reply:
Good catch. I think. How is this array used? It's not going to prevent this new alert from showing if 'personalization' has already been dismissed?

Reply:
Good question. It looks like this will only prevent personalization announcement from being displayed (and not the other way around).

In order to prevent the new announcement from being displayed, we would need to add it as a related announcement to the personalization announcement, for example.

[Instructions in the README](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/site-wide/announcements/README.md#basic-use-cases).

[Source](https://github.com/department-of-veterans-affairs/vets-website/blob/08ce05280c122c7e86827a0866853997c5e0b7f0/src/platform/site-wide/announcements/containers/Announcement.jsx#L24-L31).

Reply:
There's a readme for this stuff! Cool 😎